lib/aligning_guidelines.js

```diff
@@ -33,9 +33,13 @@ function initAligningGuidelines(canvas) {
     ctx.save();
     ctx.lineWidth = aligningLineWidth;
     ctx.strokeStyle = aligningLineColor;
+    var v = viewportTransform;
+    ctx.transform(v[0], v[1], v[2], v[3], v[4], v[5]);
```

Comment:
this will make the guidelines stroke size grow with zoom.
What is wrong in the math  down?
probably viewportTransform[4] and [5] shouldn't be multipled by zoom?

Reply:
yes, when you set the scale not equal 1, the guidelines will be draw on incorrect position. I think if the stroke size will grow with zoom, we can dynamic change the line width?

Reply:
I think we should fix the actual math to respect he matrix transformation.